mmv1/templates/terraform/examples/cloudbuild_trigger_bitbucket_server.tf.erb

```diff
@@ -0,0 +1,15 @@
+resource "google_cloudbuild_trigger" "<%= ctx[:primary_resource_id] %>" {
+  name        = "terraform-bbs-trigger"
+  location    = "us-central1"
+
+  bitbucket_server_trigger_config {
+    repo_slug = "terraform-provider-google"
+    project_key = "STAG"
+    bitbucket_server_config_resource = "projects/123456789/locations/us-central1/bitbucketServerConfigs/myBitbucketConfig"
+    push {
+        branch = "^master$"
+    }
+  }
```

Comment:
Looks like we only flex part of the changes... Could we another test and flex as many fields as possible?

Reply:
Yes, updated tests.